### PR TITLE
Range with stories unit test and docs

### PR DESCRIPTION
**Pull request by jeanpaniagua**

### CODE REVIEW

File: packages/range/src/ui-range.component.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<template>
+  <label v-if="label" :for="id" :class="theme.cssClass.label">{{
+    label
+  }}</label>
+  <input
+    type="range"
+    v-model="rangeValue"
+    :class="classes"
```
Comment (jimmymadrigal) [inline packages/range/src/ui-range.component.vue:8]: since this is only one class you can change this to :class="theme.cssClass.main"

**Review by jimmymadrigal — CHANGES_REQUESTED** [review]

**Review by jimmymadrigal — APPROVED** [review]

File: packages/range/src/docs/Range.doc.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { Story, Canvas } from '@storybook/addon-docs'
+
+# Ranges
+
+Custom input range with label.
+
+[Github repo](https://github.com/unicoderns/ui/tree/master/packages/range)
+
+```html
+    <ui-range :="args" :show="toggle" @show="show" @close="close" @hide="hide">{{ args.default }}</ui-range>
```
Comment (alejandronavarroGL) [inline packages/range/src/docs/Range.doc.mdx:10]: @jeanpaniagua I think this code should be updated with the real props.

**Review by alejandronavarroGL — CHANGES_REQUESTED** [review]
Just a minor change and we're ready to go

File: packages/range/src/docs/Range.doc.mdx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Props
+
+| Name           | Type      | Description                                                                                  |
+| -------------- | --------- | -------------------------------------------------------------------------------------------- |
+| `currentValue` | `string`  | v-model value to define the default value and get the current value                          |
+| `min`          | `number`  | start value of the range                                                                     |
+| `max`          | `number`  | max value of the range                                                                       |
+| `step`         | `number`  | By default the it would be one, but you can define a bigger interger or even a double number |
+| `disabled`     | `boolean` | allows to desable the range                                                                  |
```
Comment (alejandronavarroGL) [inline packages/range/src/docs/Range.doc.mdx:32]: typo: disable

**Review by alejandronavarroGL — APPROVED** [review]

**Pull request merged**